eve/methods/patch.py

```diff
@@ -135,8 +135,7 @@ def patch_internal(resource, payload=None, concurrency_check=False,
 
     resource_def = app.config['DOMAIN'][resource]
     schema = resource_def['schema']
-    if not skip_validation:
-        validator = app.validator(schema, resource)
+    validator = app.validator(schema, resource)
```

Comment:
It's either/or! 

My intend was to keep the ```validator``` instantiated anyways (regardless of skip_validation or not) so if it needs to be used *anywhere* in the method (even in multiple places) it would be available.

Unless instantiating it incurs a huge cost (don't think so) I feel it is cleaner to have it available throughout the method.

You have the final say, off course!